data_steward/cdr_cleaner/cleaning_rules/deid/explicit_identifier_suppression.py

```diff
@@ -0,0 +1,206 @@
+"""
+suppress all records associated with a PPI vocabulary explicit identifier associated PPI vocabulary
+The concept_ids to suppress can be determined from the vocabulary with the following regular expressions.
+
+        REGEXP_CONTAINS(concept_code, 'Signature')
+        (REGEXP_CONTAINS(concept_code, 'Name') AND REGEXP_CONTAINS(concept_code, r'(First)|(Last)|(Middle)|(Help)'))
+        (REGEXP_CONTAINS(concept_code, '(Address)|(PhoneNumber)') and 
+            NOT REGEXP_CONTAINS(concept_code, '(PIIState)|(State_)|(ZIP)') ) and concept_class_id != 'Topic'
+        REGEXP_CONTAINS(concept_code, 'SecondaryContactInfo') and concept_class_id = 'Question'
+        (REGEXP_CONTAINS(concept_code, 'SocialSecurity') and concept_class_id = 'Question')
+        and also covers all the mapped standard concepts for non standard concepts that the regex filters.
+"""
+# Python imports
+import logging
+
+# Project imports
+from resources import get_concept_id_fields
+from utils import pipeline_logging
+from common import OBSERVATION, JINJA_ENV
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+LOGGER = logging.getLogger(__name__)
+
+ISSUE_NUMBERS = ['DC1363']
+EXPLICIT_IDENTIFIER_CONCEPTS = '_explicit_records_identifier_concepts'
+
+# Creates _explicit_identifier_concepts lookup table and populates with the concept_ids that are
+# from the concept table
+LOOKUP_TABLE_CREATION_QUERY = JINJA_ENV.from_string("""
+CREATE TABLE IF NOT EXISTS `{{project_id}}.{{sandbox_dataset}}.{{lookup_table}}` AS 
+(
+WITH
+  explicit_concept_ids AS (
+  SELECT
+    *
+  FROM
+    `{{project_id}}.{{dataset_id}}.concept`
+  WHERE
+    REGEXP_CONTAINS(concept_code, 'Signature') OR((REGEXP_CONTAINS(concept_code, 'Name')
+        AND REGEXP_CONTAINS(concept_code, r'(First)|(Last)|(Middle)|(Help)'))) OR((REGEXP_CONTAINS(concept_code, '(Address)|(PhoneNumber)')
+        AND NOT REGEXP_CONTAINS(concept_code, '(PIIState)|(State_)|(ZIP)'))
+      AND concept_class_id != 'Topic') OR(REGEXP_CONTAINS(concept_code, 'SecondaryContactInfo')
+      AND concept_class_id = 'Question') OR((REGEXP_CONTAINS(concept_code, 'SocialSecurity')
+        AND concept_class_id = 'Question')))
+SELECT
+   DISTINCT *
+FROM
+  explicit_concept_ids
+UNION DISTINCT
+SELECT
+  DISTINCT *
+FROM
+  `{{project_id}}.{{dataset_id}}.concept`
+WHERE
+  concept_id IN(
+  SELECT
+    cr.concept_id_2
+  FROM
+    explicit_concept_ids AS c
+  JOIN
+    `{{project_id}}.{{dataset_id}}.concept_relationship` AS cr
+  ON
+    c.concept_id = cr.concept_id_1
+  WHERE
+    cr.relationship_id = 'Maps to')
+)
```

Comment:
nit: the following might be easier to read
```suggestion
SELECT DISTINCT
  c2.*
FROM
  explicit_concept_ids AS c
JOIN
  `{{project_id}}.{{dataset_id}}.concept_relationship` AS cr
ON
  c.concept_id = cr.concept_id_1 AND cr.relationship_id = 'Maps to'
JOIN {{project_id}}.{{dataset_id}}.concept AS c2
  cr.concept_id_2 = c2.concept_id
```